var/spack/repos/builtin/packages/redundans/package.py

```diff
@@ -11,8 +11,11 @@ class Redundans(Package):
 
     homepage = "https://github.com/Gabaldonlab/redundans"
     url      = "https://github.com/Gabaldonlab/redundans/archive/v0.13c.tar.gz"
+    git      = "https://github.com/Gabaldonlab/redundans.git"
 
-    version('0.13c', '2003fb7c70521f5e430553686fd1a594')
+    version('0.14a', commit='a20215a862aed161cbfc79df9133206156a1e9f0')
```

Comment:
I can't tell if this is actually an alpha release or if they just use a, b, c for the minor version number. They seem to have had an `alpha` release in the past. I think it's fine to put this as the preferred version.

Reply:
Without a tagged release, I think we should just add a development version pointing to the tip of the master branch.  Besides the change log & a few places in code, this is the only reference I can find to that version: https://groups.google.com/forum/#!searchin/redundans/14a%7Csort:date/redundans/XYeEt9Mys1w/Xv_goBRPAgAJ